extension/src/popup/components/signTransaction/TransactionHeader/styles.scss

```diff
@@ -0,0 +1,14 @@
+.TransactionHeader {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+
+  & > div {
+    color: var(--pal-text-tertiary);
+
+    & > :nth-child(2) {
+      color: var(--pal-text-primary);
```

Comment:
It's usually best to opt to use an SDS color when styling: https://github.com/stellar/stellar-design-system/blob/main/%40stellar/design-system/src/global.scss#L46

Our designs used the Dark Theme so you can skip down to the `dark-theme-palette`